upup/pkg/fi/nodeup/template_functions.go

```diff
@@ -281,3 +283,12 @@ func (t *templateFunctions) KubeProxyConfig() *api.KubeProxyConfig {
 
 	return config
 }
+
+func (t *templateFunctions) BuildAPIServerAnnotations() map[string]string {
+	annotations := make(map[string]string)
+	annotations["dns.alpha.kubernetes.io/internal"] = t.cluster.Spec.MasterInternalName
```

Comment:
Not by this point.  We should have populated defaults a long, long time ago, in a codepath far far away